### ISSUE TITLE
Sources for the Adobe XMP Core dependency

### ISSUE THREAD

**Issue by torakiki** [drewnoakes/metadata-extractor#588]
Hi, not sure this is the right place but I'm after the Adobe XMP core 6.1.11 dependency sources. Since it claim to be BSD there should be sources somewhere but I cannot find them, the links on the Adobe site point to a C++ implementation https://github.com/adobe/XMP-Toolkit-SDK and on the forum I got nothing useful https://community.adobe.com/t5/acrobat-sdk-discussions/xmp-metadata-open-source-library/m-p/13169904#M89861
I'm using metadata-extractor as a dependency and I'd like to make sure that all transitive dependencies are actually open source, do you maybe have some clue?

**Comment by drewnoakes** [comment]
We maintain a mirror of the source here: 

https://github.com/drewnoakes/adobe-xmp-core

It looks like the latest version of that code is 6.1.10. Indeed, if a newer release exists then Adobe should have made the source available.

**Comment by drewnoakes** [comment]
Looks like sources aren't pushed to Maven Central either: https://repo1.maven.org/maven2/com/adobe/xmp/xmpcore/6.1.11/

**Comment by torakiki** [comment]
Yes, that's the same road I took and the reason why I started asking about it in the Adobe forum

**Comment by StefanOltmann** [comment]
@torakiki I contacted Adobe several times on many ways regarding the legal aspect, but I never got a clear answer. They really have to seem given up on that.

The artifacts on Maven Central seem to be internal and come with a different license file that made me suspicious. They also seem to be somehow internal. 

On archive.org you find the old 5.1.3 which comes with a BSD license file.

That's why I based my Kotlin Multiplatform fork of on this and included this folder:
https://github.com/Ashampoo/xmpcore/tree/main/original_source

I hope that a screenshot of the original web page as well as the original source zip should be safe enough.
